gst/nnstreamer/tensor_sparse/tensor_sparse_dec.c

```diff
@@ -321,25 +326,21 @@ gst_tensor_sparse_dec_sink_event (GstPad * pad, GstObject * parent,
       gst_event_parse_caps (event, &caps);
       silent_debug_caps (self, caps, "caps");
 
+      /* set in_config */
+      structure = gst_caps_get_structure (caps, 0);
```

Comment:
In the future, with later PR, do NOT assume that 0th structure will be the only structure of gst-caps.
In general, there could be multiple structures in gst-cap and you may need to iterate.